MobileCenterUpdates/MobileCenterUpdates/MSUpdates.h

```diff
@@ -16,4 +16,11 @@
  */
 + (void)setInstallUrl:(NSString *)installUrl;
 
+/**
+ * Take a request via custom URL scheme from browser.
```

Comment:
I would rephrase it like this : "Process URL related to this module". May be triggered not only by browsers in the future. 

Also may be add a discussion saying that default integration is to place this call into the app delegate openUrl method.